client/state/google-my-business/reducer.js

```diff
@@ -26,7 +28,24 @@ const statInterval = createReducer( 'week', {
 	[ GOOGLE_MY_BUSINESS_STATS_CHANGE_INTERVAL ]: ( state, { interval } ) => interval,
 } );
 
+const stats = createReducer( null, {
+	[ GOOGLE_MY_BUSINESS_STATS_SET_DATA ]: ( state, { statType, interval, aggregation, data } ) => ( {
+		statType,
+		interval,
+		aggregation,
+		data,
+	} ),
+	[ GOOGLE_MY_BUSINESS_STATS_REQUEST ]: () => null,
+} );
+
 export default keyedReducer(
 	'siteId',
-	combineReducers( { location, statInterval: keyedReducer( 'statType', statInterval ) } )
+	combineReducers( {
+		location,
+		statInterval: keyedReducer( 'statType', statInterval ),
```

Comment:
I am moving `statsInterval` over to `ui` in #24236, so the difference between `statsInterval` and `stats` shouldn't be a problem.